.github/run-package-tests.sh

```diff
@@ -37,12 +37,15 @@ for DIR in ${DIRS}; do {
         fi
     done
 
-    echo "Running $DIR Unit Tests"
-    composer -q --no-interaction --no-ansi --no-progress update -d ${DIR};
+    echo "Installing composer in $DIR"
+    COMPOSER_ROOT_VERSION=$(cat $DIR/VERSION) composer -q --no-interaction --no-ansi --no-progress update -d ${DIR};
     if [ $? != 0 ]; then
         echo "$DIR: composer install failed" >> "${FAILED_FILE}"
+        # run again but without "-q" so we can see the error
+        COMPOSER_ROOT_VERSION=$(cat $DIR/VERSION) composer --no-interaction --no-ansi --no-progress update -d ${DIR};
         continue
```

Comment:
unrelated to this PR, but should this not be `exit` instead?

Reply:
no, because we want to run all the tests even if one fails, so we can see if the others succeed or not. By sending the error in `FAILED_FILE`, we ensure that we will exit below.